metricbeat/mb/module/module.go

```diff
@@ -24,16 +24,16 @@ const (
 )
 
 var (
-	debugf      = logp.MakeDebug("metricbeat")
+	debugf      = logp.MakeDebug("module")
 	fetchesLock = sync.Mutex{}
 	fetches     = expvar.NewMap("fetches")
 )
 
-// ModuleWrapper contains the Module and the private data associated with
+// Wrapper contains the Module and the private data associated with
 // running the Module and its MetricSets.
 //
-// Use NewModuleWrapper or NewModuleWrappers to construct new ModuleWrappers.
-type ModuleWrapper struct {
+// Use NewWrapper or NewWrappers to construct new Wrappers.
+type Wrapper struct {
```

Comment:
WDYT about renaming this file from `module.go` to `wrapper.go` and doing the same with `module_test.go`?